test/functional/simulator-e2e-specs.js

```diff
@@ -370,6 +370,28 @@ describe('advanced features', function () {
     });
   });
 
+  describe('configureLocalization', function () {
+    it(`should properly set locale settings`, async function () {
+      if (!_.isFunction(sim.configureLocalization)) {
+        return this.skip();
+      }
+
+      await sim.configureLocalization({
+        language: {
+          name: 'en'
+        },
+        locale: {
+          name: 'en_US',
+          calendar: 'gregorian',
```

Comment:
Awesome...
Btw, non-gregorian like japanese one is a typical calendar type which cause issues.